### PR TITLE
docs: Adjust the directory of Chinese documents

### LINKED ISSUE #10785: docs: Adjust the directory of Chinese documents
### Current State

The current directory is a mixture of Chinese and English. Not user friendly

### Desired State

- 快速开始
  - 入门指南
  - 配置路由
  - 负载均衡
  - 密钥验证
  - 限速
- APISIX 安装指南
- 软件架构
- 教程
  - 发布API
  - 保护API
  - 配置客户端与 APISIX 之间的双向认证（mTLS）
- APISIX 术语
  - API Gateway
  - Consumer Group
  - Global Rules
  - Plugin
  - Plugin Config
  - Route
  - Router
  - Script
  - Service
  - Upstream
  - Secret
- 插件
  - 一般插件
  - 调整请求
  - 身份认证
  - 安全防护
  - 流量控制
  - 可观测性
  - 无服务器架构
  - 其它协议
- 相关API
- 开发者
- 部署模式
- 常见问题
- 其它
- 版本发布
- 升级指南

### PR DESCRIPTION
**Pull request by smileby**
### Description

Adjust the directory of Chinese documents

Fixes #10785 

### Checklist

- [X] I have explained the need for this PR and the problem it solves
- [X] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [X] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

File: docs/zh/latest/config.json [hunk 1]
```diff
@@ -72,7 +72,7 @@
         },
         {
           "type": "category",
-          "label": "Transformation",
+          "label": "控制请求",
```
Comment (kayx23) [inline docs/zh/latest/config.json:75]: Is this the right term? Should it not be something about 转换? I'm not all that familiar with Chinese terminologies so just double checking.

Comment (smileby) [reply docs/zh/latest/config.json:75]: > Is this the right term? Should it not be something about 转换? I'm not all that familiar with Chinese terminologies so just double checking.
> 
> The other updates LGTM. Thanks.

I adjusted it to '转换请求', if there is any ambiguity, we can discuss

**Review by kayx23 — APPROVED** [review]

**Review by monkeyDluffy6017 — APPROVED** [review]

**Comment by smileby** [comment]
@kayx23  Hi，I didn’t see any updates on the official website for this submission. Is it my problem?

**Comment by kayx23** [comment]
The site is built daily and not immediately on any merges into main I believe. Give it 24 hours and see if it updates.

**Comment by yzeng25** [comment]
@smileby @kayx23  I doubt that it goes with v 3.8.x or v 3.9 release as it was merged into the master branch. Neither 3.8 version nor the Next version contains the change.



image

**Comment by smileby** [comment]
Sorry, I don't quite understand how the website directory is generated. It may be my problem, but I can help modify it. Also, please tell me how to do it.

@yzeng25  @kayx23  please help check again

**Comment by yzeng25** [comment]
> Sorry, I don't quite understand how the website directory is generated. It may be my problem, but I can help modify it. Also, please tell me how to do it.
> 
> @yzeng25 @kayx23 please help check again

Let's come back after the next release.

**Pull request merged**